src/content/components/IdleSearchField.js

```diff
@@ -20,6 +20,8 @@ class IdleSearchField extends Component {
   _timeout: number;
   _previouslyNotifiedValue: string;
 
+  props: Props;
```

Comment:
I feel like our linter should have caught the fact that this wasn't here. Nice find.